### ISSUE TITLE
Change the margin around the Focused Widget

### ISSUE THREAD

**Issue by joegeorg** [RafaelBarbosatec/tutorial_coach_mark#42]
Is there a way to change the margin around the Widget that is in focus? That is, a way to increase or decrease the space between the focused target and the TargetFocus text.

**Comment by RafaelBarbosatec** [comment]
Hi @joegeorg !

You can configure this with the `paddingFocus` parameter.
It is not yet possible to configure this specifically for each focus

```
 void showTutorial() {
    tutorialCoachMark = TutorialCoachMark(context,
        targets: targets,
        colorShadow: Colors.red,
        textSkip: "SKIP",
        paddingFocus: 10,
        opacityShadow: 0.8, onFinish: () {
      print("finish");
    }, onClickTarget: (target) {
      print(target);
    }, onClickSkip: () {
      print("skip");
    })
      ..show();
  }
```

**Comment by RafaelBarbosatec** [comment]
Available in version 0.5.2
